modules/@apostrophecms/template/views/outerLayoutBase.html

```diff
@@ -22,6 +22,7 @@
     {% endblock %}
     {% block apostropheContextMenu %}
       <div id="apos-context-menu"></div>
+      <div id="apos-notification"></div>
```

Comment:
Should we rename this block if it will have more than the context menu in it?

Reply:
Not sure this is the right place for notifications. I placed it here randomly..

Reply:
@boutell What do you think? Should we make this block `apostropheUserInterface` or something like that?

Reply:
Does the location in the DOM actually matter, to either of these apps? (I honestly don't know as I'm not the one writin' styles here.)

Reply:
It shouldn't. The question is more if we should keep the block as `apostropheContextMenu` and move the notification one somewhere else, or rename it to hold multiple UI placeholders.

Reply:
Done